src/workspace-data/data-table/entity-service/EntitiesContent.test.ts

```diff
@@ -621,6 +621,39 @@ describe('IGV & Workflow Icons and Tool Drawer', () => {
     expect(igvButton).toHaveAttribute('aria-disabled', 'true');
   });
 
+  it('renders enabled IGV button even when no entities selected', async () => {
```

Comment:
I think maybe I forgot to commit this test in the last PR? So not relevant to this PR but still relevant to the overall ticket.